tests/storage/blockvolume_test.py

```diff
@@ -71,9 +104,9 @@ class TestBlockVolumeSize(VdsmTestCase):
         [(sc.PREALLOCATED_VOL, sc.COW_FORMAT, GiB, None), GiB],
         # Sparse, cow, capacity config.volume_utilization_chunk_mb - 1,
         # No initial size.
-        # Expected 1024 MiB allocated (config.volume_utilization_chunk_mb)
-        [(sc.SPARSE_VOL, sc.COW_FORMAT, (CONFIG.getint(
-            "irs", "volume_utilization_chunk_mb") - 1) * MiB, None), GiB],
+        # Expected 1023 MiB allocated (config.volume_utilization_chunk_mb - 1)
```

Comment:
Not sure whether the comment should be updated (in lines 105 & 107): `config.volume_utilization_chunk_mb` --> `CHUNK_SIZE_MB`.

Reply:
Both are true, and leaving the config in the comment hints for the source of the `CHUNK_SIZE_MB`, so I think we can keep it as is?